bodhi/server/__init__.py

```diff
@@ -146,7 +146,7 @@ def initialize_db(config):
     """
     #: The SQLAlchemy database engine. This is constructed using the value of
     #: ``DB_URL`` in :data:`config``.
-    engine = engine_from_config(config, 'sqlalchemy.')
+    engine = engine_from_config(config.copy(), 'sqlalchemy.')
```

Comment:
I assume this is here to prevent sqlalchemy from modifying the config (so the pop doesn't change it)? It might be good to put a comment to that effect here for future readers.